merkle/coniks/coniks.go

```diff
@@ -60,11 +62,13 @@ func (m *hasher) EmptyRoot() []byte {
 func (m *hasher) HashEmpty(treeID int64, index []byte, height int) []byte {
 	depth := m.BitLen() - height
 
+	buf := new(bytes.Buffer)
```

Comment:
Do you think it would be worth preallocating the buffer to avoid any runtime resizing?

Reply:
Probably only if it resizes more than once, which is a possibility. I can test that with a benchmark.

Reply:
Seems about 5% faster. Might be worth it though the time in this function is probably way less than in the hashing.